docs/src/pages/components/badges/UnstyledBadge.js

```diff
@@ -74,13 +72,15 @@ function BadgeContent() {
 
 export default function UnstyledBadge() {
   return (
-    <Stack spacing={4} direction="row">
+    <Box
+      sx={{ '& > :not(style) + :not(style)': { ml: (theme) => theme.spacing(4) } }}
```

Comment:
Only one thing here, why not just `& > * + *`

Reply:
@siriwatknp Two reasons: 

1. to increase the specificity

<img width="581" alt="Capture d’écran 2021-06-22 à 15 05 07" src="https://user-images.githubusercontent.com/3165635/122929500-4f47d080-d36b-11eb-91b9-c90ad1ba6a7b.png">
<img width="313" alt="Capture d’écran 2021-06-22 à 15 05 00" src="https://user-images.githubusercontent.com/3165635/122929498-4e16a380-d36b-11eb-996b-bf29e289f7c1.png">

2. emotion adds <style> element in the body when the SSR is not configured correctly. Al

Reply:
This would work, right? 

```suggestion
      sx={{ '& > :not(style) + :not(style)': { ml: 4 } }}
```

Reply:
yes!